### ISSUE TITLE
Aardvark: Foxx service configuration values are being stored in escaped form

### ISSUE THREAD

**Issue by sinfex** [arangodb/arangodb#2161]
I'm using the latest ArangoDB of the respective release series: 3.0.10
On this operating system: CentOS Linux 7.2
Foxx service mode: Development

Manifest:
```
"configuration": {
	"option": {}
}
```

With the `option` setting set to `' & "` in the web interface:

```
module.context.configuration.option === "&#39; &amp; &quot;"
```

**Comment by hkernbach** [comment]
Thanks. Fixed in devel and 3.0.

**Comment by pluma** [comment]
@hkernbach what about 3.1?

**Comment by hkernbach** [comment]
My mistake. Fix is included in devel and 3.1 - 3.0 was missing. Corrected that.
